### PR TITLE
update MDS course descriptions and 2022-23 instructors

### PR DESCRIPTION
**Pull request by zmx721**

### CODE REVIEW

**Review by kvarada — COMMENTED** [review]
What's the purpose of having two columns for description? I think "Expanded Description" column should be enough? I would change it to just Description if we are only keeping one column.

File: descriptions.md [hunk 1]
```diff
@@ -4,30 +4,30 @@ title: MDS Courses
...
+Course Number  |  Block  |  Course Title                                |  Short Description     |Expanded Description| 2022-23 Lecture Instructor | 2022-23 Lab Instructor | 
+---------------|---------|----------------------------------------------|------------------------|------------|------------|------------|
+[DSCI 511](https://github.com/UBC-MDS/DSCI_511_prog-dsci)       |  1      |  Programming for Data Science              |  Pseudo-code. Program design and structure. Flow control. Iteration. Lists (arrays). Functions. File I/O. Classes, objects, methods, and libraries.  |  Program design and data manipulation with Python. Overview of data structures, iteration, flow control, and program design relevant to data exploration and analysis. When and how to exploit pre-existing libraries. | [Arman Ahmadi](https://www.linkedin.com/in/arman-s-ahmadi-38983864/) | [Arman Ahmadi](https://www.linkedin.com/in/arman-s-ahmadi-38983864/)
```
Comment (armansa1) [inline descriptions.md]:
```suggestion
[DSCI 511](https://github.com/UBC-MDS/DSCI_511_prog-dsci)       |  1      |  Programming for Data Science              |  Pseudo-code. Program design and structure. Flow control. Iteration. Lists (arrays). Functions. File I/O. Classes, objects, methods, and libraries.  |  Program design and data manipulation with Python. Overview of data structures, iteration, flow control, and program design relevant to data exploration and analysis. When and how to exploit pre-existing libraries. | [Arman Seyed-Ahmadi](https://www.linkedin.com/in/arman-s-ahmadi-38983864/) | [Arman Seyed-Ahmadi](https://www.linkedin.com/in/arman-s-ahmadi-38983864/)
```

File: descriptions.md [hunk 2]
```diff
@@ -4,30 +4,30 @@ title: MDS Courses
...
+[DSCI 512](https://github.com/UBC-MDS/DSCI_512_alg-data-struct)       |  2      |  Algorithms and Data Structures              | Basic algorithms. Recursion. Data structures including linked lists, queues, stacks, trees, graphs, and hash tables. Searching and sorting. Introduction to complexity including Big-O notation, efficiency, and scalability.  |  How to choose and use appropriate algorithms and data structures to help solve data science problems. Key concepts such as recursion and algorithmic complexity (e.g., efficiency, scalability). | [Arman Ahmadi](https://www.linkedin.com/in/arman-s-ahmadi-38983864/) | [Arman Ahmadi](https://www.linkedin.com/in/arman-s-ahmadi-38983864/)
```
Comment (armansa1) [inline descriptions.md]:
```suggestion
[DSCI 512](https://github.com/UBC-MDS/DSCI_512_alg-data-struct)       |  2      |  Algorithms and Data Structures              | Basic algorithms. Recursion. Data structures including linked lists, queues, stacks, trees, graphs, and hash tables. Searching and sorting. Introduction to complexity including Big-O notation, efficiency, and scalability.  |  How to choose and use appropriate algorithms and data structures to help solve data science problems. Key concepts such as recursion and algorithmic complexity (e.g., efficiency, scalability). | [Arman Seyed-Ahmadi](https://www.linkedin.com/in/arman-s-ahmadi-38983864/) | [Arman Seyed-Ahmadi](https://www.linkedin.com/in/arman-s-ahmadi-38983864/)
```

File: descriptions.md [hunk 3]
```diff
@@ -4,30 +4,30 @@ title: MDS Courses
...
+[DSCI 552](https://github.com/UBC-MDS/DSCI_552_stat-inf-1)       |  2      |  Statistical Inference and Computation I     |  Random variables, parameters, observed data, statistics (distinctions and connections). Estimation: point and interval. Two-group comparisons, frequentist version. Simulation-based approaches.  |  The statistical and probabilistic foundations of inference. Large sample results. The frequentist paradigm. | [Alexi Rodríguez-Arelis](https://alexrod.netlify.app/)  | [Alexi Rodríguez-Arelis](https://alexrod.netlify.app/)
+[DSCI 513](https://github.com/UBC-MDS/DSCI_513_database-data-retr)       |  3      |  Databases and Data Retrieval              |  Relational schemas. SQL queries. Database programming using embedded SQL. XML and XQuery.  |  How to work with data stored in relational database systems. Storage structures and schemas, data relationships, and ways to query and aggregate such data. | [Arman Ahmadi](https://www.linkedin.com/in/arman-s-ahmadi-38983864/)  | [Arman Ahmadi](https://www.linkedin.com/in/arman-s-ahmadi-38983864/)
```
Comment (armansa1) [inline descriptions.md]:
```suggestion
[DSCI 513](https://github.com/UBC-MDS/DSCI_513_database-data-retr)       |  3      |  Databases and Data Retrieval              |  Relational schemas. SQL queries. Database programming using embedded SQL. XML and XQuery.  |  How to work with data stored in relational database systems. Storage structures and schemas, data relationships, and ways to query and aggregate such data. | [Arman Seyed-Ahmadi](https://www.linkedin.com/in/arman-s-ahmadi-38983864/)  | [Arman Seyed-Ahmadi](https://www.linkedin.com/in/arman-s-ahmadi-38983864/)
```

File: descriptions.md [hunk 4]
```diff
@@ -4,30 +4,30 @@ title: MDS Courses
...
+[DSCI 573](https://github.com/UBC-MDS/DSCI_573_feat-model-select)       |  3     |  Feature and Model Selection                |  Performance of a classification model. Generalization error, overfitting of training data. Shrinkage, feature selection, Akaike Information Criterion, Bayesian Information Criterion. k-fold cross validation. Bootstrapping. Receiver Operating Characteristic curve. Elastic nets, regularization.  |  How to evaluate and select features and models. Cross-validation, ROC curves, feature engineering, and regularization.     | [Varada Kolhatkar](https://kvarada.github.io/) | [Joel Östblom](https://joelostblom.com/)
+[DSCI 572](https://github.com/UBC-MDS/DSCI_572_sup-learn-2)       |  4      |  Supervised Learning II                   |  Support Vector Machines. Random Forests. Ensemble Classifiers. Graphical models.  |  Introduction to numerical optimization (e.g., gradient descent). Neural networks and deep learning. | [Arman Ahmadi](https://www.linkedin.com/in/arman-s-ahmadi-38983864/) | [Arman Ahmadi](https://www.linkedin.com/in/arman-s-ahmadi-38983864/)
```
Comment (armansa1) [inline descriptions.md]:
```suggestion
[DSCI 572](https://github.com/UBC-MDS/DSCI_572_sup-learn-2)       |  4      |  Supervised Learning II                   |  Support Vector Machines. Random Forests. Ensemble Classifiers. Graphical models.  |  Introduction to numerical optimization (e.g., gradient descent). Neural networks and deep learning. | [Arman Seyed-Ahmadi](https://www.linkedin.com/in/arman-s-ahmadi-38983864/) | [Arman Seyed-Ahmadi](https://www.linkedin.com/in/arman-s-ahmadi-38983864/)
```

**Pull request merged**